tests/test_fyle/test_task.py

```
@@ -369,3 +374,819 @@
+    expense.fund_source = 'PERSONAL'
+    expense.save()
+
+    mocker.patch('apps.fyle.tasks.get_fund_source', side_effect=['PERSONAL', 'CCC'])
+    mock_handler = mocker.patch('apps.fyle.tasks.handle_fund_source_changes_for_expense_ids')
+
+    expense_objects = [{
+        'id': expense.expense_id,
+        'fund_source': 'CCC'
+    }]
+
+    update_non_exported_expenses(expense_objects, expense.workspace_id)
```

Pylint (3.3.7)

[error] Too many positional arguments for function call

(E1121)